AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -81,8 +81,8 @@ foromtb.com,dekudeals.com,encuentra24.com,krone.at##body div.fc-consent-root
 jneurosci.org,medrxiv.org,novartis.com.tr,royal.uk,transcom.com,netcologne.de,belltower.news,bigfm.de,click.ro,demokrata.hu,emma.de,gysev.hu,humanite.fr,ikgastarten.nl,la7.it,london.gov.uk,mavcsoport.hu,mcdonalds.at,mcdonalds.it,mises.org,mnm.hu,musescore.com,nbcsports.com,pnas.org###sliding-popup
 ! Sourcepoint cookie consent
 theguardian.com##div[id^="sp_message_"]
-preis.de,vorname.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com,motorcyclenews.com,techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#div[id^="sp_message_"] { display: none !important; }
-faz.net,chip.de,preis.de,giga.de,motorcyclenews.com,finanzen.net,vorname.com,jezebel.com,theonion.com,theinventory.com,thetakeout.com,theroot.com,avclub.com,deadspin.com,lifehacker.com,computerbild.de,techbook.de,lvz.de,sueddeutsche.de,gutefrage.net,shieldsgazette.com,spieletipps.de,ekstrabladet.dk,focus.de,demorgen.be,volkskrant.nl,trouw.nl,parool.nl,techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#html.sp-message-open { overflow: visible !important; position: static !important; margin-top: 0!important; }
+pcworld.com,preis.de,vorname.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com,motorcyclenews.com,techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#div[id^="sp_message_"] { display: none !important; }
+pcworld.com,faz.net,chip.de,preis.de,giga.de,motorcyclenews.com,finanzen.net,vorname.com,jezebel.com,theonion.com,theinventory.com,thetakeout.com,theroot.com,avclub.com,deadspin.com,lifehacker.com,computerbild.de,techbook.de,lvz.de,sueddeutsche.de,gutefrage.net,shieldsgazette.com,spieletipps.de,ekstrabladet.dk,focus.de,demorgen.be,volkskrant.nl,trouw.nl,parool.nl,techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#html.sp-message-open { overflow: visible !important; position: static !important; margin-top: 0!important; }
```

Comment:
Video is broken https://www.pcworld.com/video/106661/how-to-use-windows-security-in-windows-10

Reply:
Video works fine on my end, i dont see anything common with rules on this video.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/107024641-59e93b00-67b9-11eb-950d-28bb60b8b7fa.png)
</details><br/>

Reply:
Have you checked it under VPN? Look what I've got
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/107025184-1c38e200-67ba-11eb-940a-976e309e3814.png)

</details><br/>

Reply:
Video also won't load with VPN on and AG off

Reply:
After accepting cookies manually
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/107026635-0d532f00-67bc-11eb-963d-07fb1352616f.png)

</details><br/>

Reply:
@AdamWr @BlazDT please check it on your end.

Reply:
On my end video works fine only if I accept cookies.

Reply:
I've cleared my cookies and cache and video still work on my side for some reason =\
Should i remove these rules and add clicker tweak instead?

Reply:
Of course.